docs/apis-tools/go-client/go-get-started.md

```diff
@@ -7,6 +7,12 @@ sidebar_label: "Getting started with the Go client"
 import Tabs from "@theme/Tabs";
 import TabItem from "@theme/TabItem";
 
+:::caution
```

Comment:
Should this perhaps be added to the current `8.5` docs, and should we remove all references of the Go client in the `next`, `8.6` docs?

Reply:
CC @megglos